code/julia/Javascript/lab01_A/part_A.js

```diff
@@ -0,0 +1,20 @@
+// Unit converter
```

Comment:
**Needs Work**
* Entered `6` and `ft` and `inches`:
    * Expected about 72 inches.
    * Result is `1.82 inches`.
![image](https://user-images.githubusercontent.com/47562501/204383730-a346a561-3125-4644-83a9-3ea055158ba2.png)